src/components/TabletApp/RightSide/QuickBook/QuickBook.tsx

```diff
@@ -137,27 +218,28 @@ const QuickBook = () => {
                                         }
                                     }}
                                     onClose={() => setAutoComplete(false)}
+                                    onChange={handleChange}
                                     disablePortal
                                     id="combo-box-demo"
                                     options={owners}
                                     sx={{ width: 300 }}
-                                    value={ownerName}
+                                    value={
+                                        owner === undefined ? "" : owner.name
+                                    }
                                     renderInput={(params) => (
-                                        <TextField
-                                            {...params}
-                                            label="Participants"
-                                            value={ownerName}
-                                            onChange={handleChange}
-                                        />
+                                        <TextField {...params} label="Owner" />
                                     )}
                                 />
                             </Box>
                             <Box
                                 display="flex"
                                 justifyContent="center"
-                                onClick={handleSearchOwner}
+                                onClick={handleSubmitOwner}
```

Comment:
the "onClick" method should be on <Button /> and not on <Box/>

Reply:
Moved the onClick method to the button instead of the Box component